### ISSUE TITLE
an acquisition problem

### ISSUE THREAD

**Issue by hjllsd** [Bannerets/tdl#133]
Thank you for viewing, looking forward to your answer

then question
How to get group message link
A link in the following format https://t.me/xxxxxx/857164
How can I get this id and group name
Thanks and looking forward to your answer

**Comment by Bannerets** [comment]
Use [`getMessageLink`](https://core.telegram.org/tdlib/docs/classtd_1_1td__api_1_1get_message_link.html)

```javascript
await client.invoke({
  _: 'getMessageLink',
  chat_id: 00000,
  message_id: 11111
})
````

**Comment by hjllsd** [comment]
first of all thank you for your answer
I'm already using this method, thanks again for your answer

**Comment by Bannerets** [comment]
so the problem is solved?

**Comment by hjllsd** [comment]
No, I used this method after seeing your reply, but I don’t seem to find the message_id parameter in updateChatLastMessage, so is it in other methods?

**Comment by Bannerets** [comment]
it's in `updateChatLastMessage.last_message.id` (`last_message` can be null, in this case you can't access the message)

**Comment by hjllsd** [comment]
Thank you for your enthusiastic answer. The problem should be solved. If it is solved, I will come back and record it.
Thank you again for your enthusiastic answer

**Comment by hjllsd** [comment]
Problem solved, thank you again
